deps/db/src/logseq/db/frontend/entity_plus.cljs

```diff
@@ -1,8 +1,8 @@
-(ns frontend.db.datascript.entity-plus
+(ns logseq.db.frontend.entity-plus
```

Comment:
Moved here so that it can be used by more deps.

Reply:
Fwiw this isn't something we can use in nbb/sci yet because sci doesn't yet have access to the native protocols - https://github.com/babashka/sci/issues/639. For now I'll make this cljc, enable it for cljs and disable for nbb. From the git history I read, ldb/get-block-and-children, db-content/update-block-content and most fns in outliner.tree are the only deps fns that use this ns. None of these fns effect critical nbb fns. If you're aware of other deps fns that use this ns, please let me know